controlplane/clickhouse/migrations/20230910193138_traces_by_operation_quarter_hourly_mv.sql

```diff
@@ -0,0 +1,53 @@
+-- migrate:up
+
+CREATE MATERIALIZED VIEW IF NOT EXISTS cosmo.traces_by_operation_quarter_hourly_mv (
+   Timestamp DateTime('UTC') CODEC (Delta(4), ZSTD(1)),
+   OperationName String CODEC (ZSTD(1)),
+   OperationType String CODEC (ZSTD(1)),
+   FederatedGraphID String CODEC(ZSTD(1)),
+   OrganizationID String CODEC(ZSTD(1)),
+   Subscription Bool CODEC(ZSTD(1)),
+   TotalRequests UInt64 CODEC(ZSTD(1)),
+   TotalRequestsError UInt64 CODEC(ZSTD(1)),
+   TotalRequestsOk UInt64 CODEC(ZSTD(1)),
+   DurationQuantiles AggregateFunction(quantiles(0.75, 0.9, 0.95, 0.99), Int64) CODEC(ZSTD(1)),
+   LastCalled DateTime('UTC') CODEC (ZSTD(1))
+)
+ENGINE = SummingMergeTree
+PARTITION BY toDate(Timestamp)
+ORDER BY (
```

Comment:
Have read about the importancy about the order by keys? The order is important. I'd recommend to put the OrganizationID before the federatedGraphID.

https://clickhouse.com/docs/en/optimize/sparse-primary-indexes

Reply:
Done